packages/vscode-extension/package.json

```diff
@@ -36,9 +36,15 @@
   "contributes": {
     "commands": [
       {
-        "command": "neo4j.manageConnection",
+        "command": "neo4j.editConnection",
         "category": "Neo4j",
-        "title": "Manage Connection",
+        "title": "Edit connection",
```

Comment:
From a call with Greg, I got the feedback we should be renaming the `Manage Connection` to `Edit connection`

Reply:
And also that we should be showing `Connection` in lower case in all of the menus, hence why in many places this PR rewrites that